tests/common/test_version.py

```diff
@@ -78,6 +85,7 @@ def test_distro_is_correct_format_when_openbsd(self, platform_system_name, mock_
         self.assertListEqual(openbsd_list, osinfo)
         return
 
+    @skip_if_predicate_false(is_platform_supported, "platform.dist() is deprecated in Python 3.7+")
```

Comment:
Instead of skipping these tests shouldn't we fix them to test using the distro module instead? (Since we dont have the Canonical deadline anymore might as well do it right)

Reply:
or should we have versions for platform and distro?

Reply:
Updating them to cover both scenarios: when platform.dist is supported and when it isn't.